dashboard/test/ui/step_definitions/gamelab.rb

```diff
@@ -0,0 +1,21 @@
+# Gamelab-specific cucumber step definitions
+
+# Which stage of allthethings.script contains the gamelab levels; this way we
+# only have to update in one place if this changes.
+GAMELAB_ALLTHETHINGS_STAGE = 19
+
+Given /^I start a new Game ?Lab project$/ do
+  steps <<-STEPS
+    And I am on "http://learn.code.org/projects/gamelab/new"
+    And I rotate to landscape
```

Comment:
Wonder if "I rotate to landscape" should just be part of our "I am on" step (though no need for you to change here)